js/default_presets/airplane_tail.js

```diff
@@ -0,0 +1,178 @@
+window.default_preset_airplane_tail = {
```

Comment:
Idea is great, but let's do it in a much better way!
1. No more assigning anything to `window` object, this is so 90s
2. Let's store definitions in an array and push new elements as we go

so it would look like

```
var presetDefinitions = presetDefinitions || [];

presetDefinitions.push({
//here comes the preset object
})

```

Reply:
I think @michellgai tried this first when we did this but it did not work out. I don't know the exact reason. But I'll ask him, maybe I'm wrong here.